include/configs/piksiv3_dev.h

```diff
@@ -45,28 +45,29 @@
 
 /* CPU clock */
 #ifndef CONFIG_CPU_FREQ_HZ
-# define CONFIG_CPU_FREQ_HZ 800000000
+# define CONFIG_CPU_FREQ_HZ              800000000
 #endif
 
 /* Cache options */
 #define CONFIG_CMD_CACHE
-#define CONFIG_SYS_CACHELINE_SIZE 32
+#define CONFIG_SYS_CACHELINE_SIZE        32
 
 #define CONFIG_SYS_L2CACHE_OFF
 #ifndef CONFIG_SYS_L2CACHE_OFF
```

Comment:
is this `ifndef` a no-op?

Reply:
Yes, would need to see if `CONFIG_SYS_L2CACHE_OFF` is used elsewhere, or it may be in this form so it can be easily commented out.